container/docker/factory.go

```diff
@@ -40,6 +41,7 @@ var ArgDockerEndpoint = flag.String("docker", "unix:///var/run/docker.sock", "do
 var DockerNamespace = "docker"
 
 // Basepath to all container specific information that libcontainer stores.
+// TODO: Deprecate this flag
```

Comment:
Should we change the help documentation to state that this is deprecated and setting it has no effect?  Maybe "deprecated (no-op): docker root directory is now read from docker info"